pkg/atom/templates/crd.yml.tmpl

```diff
@@ -18,10 +18,13 @@ spec:
           properties:
             started:
               type: string
+              nullable: true
             status:
               type: string
+              nullable: true
             spec:
               type: object
+              nullable: true
```

Comment:
In the [Apply](https://github.com/convox/convox/blob/b4650f7b0cc941fd44ebc0ca6baed956db0cd851/pkg/atom/atom.go#L138) function, on the `atom` package, was trying to create an empty resource and it was failing because in the schema the tags weren't defined as nullable.

```
                 a, err = c.Atom.AtomV1().Atoms(ns).Create(&aa.Atom{
			ObjectMeta: am.ObjectMeta{
				Name: name,
			},
		})
		if err != nil {
			return errors.WithStack(err)
		}
```